src/EditorFeatures/CSharpTest/ExtractMethod/ExtractMethodTests.LanguageInteraction.cs

```diff
@@ -1113,7 +1113,7 @@ public static void Main()
                         p1 = NewMethod(p2);
                     }
 
-                    private static object NewMethod(object p2)
+                    private static global::System.Object NewMethod(global::System.Object p2)
```

Comment:
intentional change.  EM now calls trhough the correct syntax generator code.  but the tests don't actually run the simplification pass (since the tests unfortunately test parts of the api directly).